### ISSUE TITLE
About cross validation

### ISSUE THREAD

**Issue by HiYKY** [Jess2110/Epileptic_Seizure_Detection_ML#1]
Hi, you use X_train and y_train to fit the model (Line 96) and X_test and y_test to evaluate the model (Line 98). 

However, X_train, X_test, y_train, y_test are split by train_test_split already at line 37.

Therefore, the model is just trained for 5 times without using train set and validation set.

Would you like to consider this solution at line 96 and line 98?

history=model.fit(X[train], y[train], validation_split=0.33,epochs=15, batch_size=10, verbose=0)

scores = model.evaluate(X[test], y[test], verbose=0)
